specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-04-02-preview/managedClusters.json

```diff
@@ -6177,6 +6398,152 @@
     "CapacityReservationGroupID": {
       "description": "Capacity Reservation Group ID for AgentPool to associate",
       "type": "string"
+    },
+    "TrustedAccessRoleRule": {
+      "type": "object",
+      "description": "Rule for trusted access role",
```

Comment:
I think we should mention that this corresponds to an actual Kubernetes Role in the cluster, and link to the Kubernetes role documentation (so that people can understand what the options here mean, since they just parallel the k8s one)

Reply:
Or maybe we should put that link on `TrustedAccessRole` and not `RoleRule`. I think it's probably worth having though?

Reply:
added on rules property